app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -9,7 +22,7 @@ export default function NoteTable({
 }) {
   return (
     <div className="w-full px-4">
-      <table className="nui-table nui-table--checkbox">
+      <table className="nui-table nui-table--checkbox table-fixed">
```

Comment:
Why do we need `table-fixed` class here?

Reply:
I added it to have fixed width columns for truncate to work because I didn't add the max-width constraint there.
But now, I fixed it